### ISSUE TITLE
Could not install operator via Helm following the documentation - Helm returns 404

### ISSUE THREAD

**Issue by lusu007** [CrunchyData/postgres-operator#3504]
Please ensure you do the following when reporting a bug:

- [x] Provide a concise description of what the bug is.
- [x] Provide information about your environment.
- [x] Provide clear steps to reproduce the bug.
- [x] Attach applicable logs. Please do not attach screenshots showing logs unless you are unable to copy and paste the log data.
- [x] Ensure any code / output examples are [properly formatted](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code) for legibility.

Note that some logs needed to troubleshoot may be found in the `/pgdata/<CLUSTERNAME>/pg_log` directory on your Postgres instance.

An incomplete bug report can lead to delays in resolving the issue or the closing of a ticket, so please be as detailed as possible.

If you are looking for [general support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/), please view the [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) page for where you can ask questions.

Thanks for reporting the issue, we're looking forward to helping you!

## Overview

I would like to use the operator to administer multiple Postgres clusters and have followed the documentation first. I want to [install the operator via Helm chart](https://access.crunchydata.com/documentation/postgres-operator/v5/installation/helm/) (Section: Installing directly from the registry). However, Helm returns an error when following the commands shown in the documentation. It looks like the Helm chart does not exist in the OCI registry.

## Environment

Please provide the following details:

- Platform: Kubernetes
- Platform Version: 1.25.4
- PGO Image Tag: -
- Postgres Version: -
- Storage: -

### REPRO

Provide steps to get to the error condition:

1. Run `helm install pgo oci://registry.developers.crunchydata.com/crunchydata/pgo`

### EXPECTED

1. Helm should install the chart.

### ACTUAL

1. Helm throws an 404 error

## Logs

```
Error: INSTALLATION FAILED: GET "https://registry.developers.crunchydata.com/v2/crunchydata/pgo/tags/list": unexpected status code 404: 404: This container version is no longer available from the Crunchy Data Developer Program. For information on accessing these containers, please contact info@crunchydata.com.
```

**Comment by sanju7216** [comment]
We are seeing similar issue using operatorhub, is there a workaround or any fix for this??
looks like `registry.connect.redhat.com/crunchydata/postgres-operator@sha256:4ad944f4f01a85249aa7678521e61fe48ed150ac43f3c38f33b42f7a32eb9961` is unavailable


```
Failed to pull image "registry.connect.redhat.com/crunchydata/postgres-operator@sha256:4ad944f4f01a85249aa7678521e61fe48ed150ac43f3c38f33b42f7a32eb9961": rpc error: code = Unknown desc = reading manifest sha256:4ad944f4f01a85249aa7678521e61fe48ed150ac43f3c38f33b42f7a32eb9961 in registry.connect.redhat.com/crunchydata/postgres-operator: manifest unknown: manifest unknown
```

<img width="1269" alt="Screenshot 2022-12-19 at 12 24 02" src="https://user-images.githubusercontent.com/728940/208425493-7fb880ab-aa65-4819-940e-d96e0e48c5d3.png">
<img width="963" alt="Screenshot 2022-12-19 at 12 25 24" src="https://user-images.githubusercontent.com/728940/208425714-b06b2484-667c-4104-b1b9-f0266e6b8c0a.png">

**Comment by yuvraj-vansure** [comment]
![image](https://user-images.githubusercontent.com/81155309/208449855-4cf5e273-798e-46c7-ae04-f0e124bb6ce1.png)

image mentioned in the above issue is available on [catalog.redhat.com](https://catalog.redhat.com/software/containers/crunchydata/postgres-operator/5c78066cecb5240adfaaeaa3?tag=ubi8-5.2.0-0&push_date=1662744469000&container-tabs=gti) but still we cannot pull this image 
Also it mentions about newer version _5.3.0_, but it is still not available on certified-operators catalog source. when can we expect release of _5.3.0_

**Comment by benjaminjb** [comment]
@lusu007 Can you retry installing through the OCI registry? I believe the OCI was only fully setup recently, possibly after your first attempt. I was able to install a working PGO instance using the command you gave.

**Comment by benjaminjb** [comment]
@yuvraj-vansure A quick note while I look into the 5.2.0 issue that you mention: we are working with Red Hat on making 5.3.0 available on certified-operators.

**Comment by benjaminjb** [comment]
@sanju7216 and @yuvraj-vansure : I can confirm that the 5.2.0-0 image is visible in the catalog, but unable to be retrieved. I'm looking into it with Red Hat, as that is the only image that's a problem right now.

I can also confirm that image 5.2.1-1 was accessible -- I mean, I was at least able to `docker pull` it from Red Hat. Until we can work out this issue with Red Hat about the 5.2.0 image, you might work around by manually switching to image 5.2.1-1. Hope that helps!

**Comment by chicha75** [comment]
> @sanju7216 and @yuvraj-vansure : I can confirm that the 5.2.0-0 image is visible in the catalog, but unable to be retrieved. I'm looking into it with Red Hat, as that is the only image that's a problem right now.
> 
Any new about this from redhat ? I'm trying to install ibm mas which depends from the crunchy operator and I can't figure out how to specify a different version of the operator from the ibm mas installer.. So My installation has been stuck for more than 2 weeks now :-|

**Comment by lusu007** [comment]
> 1. helm install pgo oci://registry.developers.crunchydata.com/crunchydata/pgo

Sorry for my late answer. I tried installing the operator via the OCI registry and it seems to work now.

In my opinion, next time the system should be configured properly before the installation possibility is added to the documentation. Otherwise, it causes unnecessary confusion.

Thank you for your work.

**Comment by chicha75** [comment]
I confirm that the 5.3.0 version of the operator is (still?) not available on ocp 4.8. This is a blocking issue for us.
Is there any reason it's available on ocp 4.10 and not on ocp 4.8 ?

**Comment by andrewlecuyer** [comment]
We are working with Red Hat to fix the issues around the 5.2.0 image being unavailable through their registry.  And considering the original issue involving Helm OCI installation has been resolved, I am proceeding with closing this issue.

For more information and/or to discuss your needs further, please feel free to reach out at info@crunchydata.com.

**Comment by ValClarkson** [comment]
The issue with the 5.2.0 image has now been fixed in the Red Hat Connect portal, Please retry pulling the image.
